audiodecoder.usf/platforms.txt

```diff
@@ -0,0 +1 @@
+all
```

Comment:
This currently doesn't succeed for android and ios. Do you plan to fix it?

Reply:
i don't think i can, i think there are some x86 assembly in there.

Reply:
from a quick look it looks like it is caused by https://github.com/xbmc/audiodecoder.usf/blob/master/lib/lazyusf/CMakeLists.txt#L33-L39 which should be fixable.

Reply:
okay, then let's try that.